assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -133,7 +133,7 @@ angular.module('openshiftConsole')
             );
             return d.promise;
           });
-          $location.path("project/" + $scope.projectName + "/overview");
+          Navigate.toProjectOverview($scope.projectName);
```

Comment:
would rather we had a service that knew how to generate URLs and return them, there will be cases where we need the URLs and won't necessarily be wanting to navigate right then.